kokoro/builds/upload_symbols.sh

```diff
@@ -54,6 +54,9 @@ function install_breakpad_tools {
 }
 
 function upload_symbol_file {
+  # Fail on any error.
+  set -e
```

Comment:
I'm not sure if that's exactly what we want. The `set -e` is not a local thing. It will leak out of the function and stay active in the context of the caller after the function has been executed. I assume it's currently not set in the context of the caller otherwise it wouldn't be necessary.

If it doesn't lead to problems I would prefer to set it at the beginning of build.sh.

If it leads to problems we need to make sure that it doesn't leak. Executing the function in a subshell should work:

```
function upload_symbol_file (
  set -e
  local api_key=$1
  ...
)
```

(parens instead of curly braces will make the function execute in a subshell and the `set -e` should only affect the subshell.)

The same applies to the other function below.

Reply:
That's a good point. I was under the impression, that `set -e` was already set globally, which was wrong. It was explicitly disabled for the call of upload symbols. I fixed that on the call site.

Reply:
Even better. Thanks!